src/braket/tasks/local_quantum_task_batch.py

```diff
@@ -29,16 +36,112 @@ class LocalQuantumTaskBatch(QuantumTaskBatch):
 
     def __init__(
         self,
-        results: List[
-            Union[
-                GateModelQuantumTaskResult,
-                AnnealingQuantumTaskResult,
-                PhotonicModelQuantumTaskResult,
+        results: Optional[
+            List[
+                Union[
+                    GateModelQuantumTaskResult,
+                    AnnealingQuantumTaskResult,
+                    PhotonicModelQuantumTaskResult,
+                ]
             ]
-        ],
+        ] = None,
     ):
         self._results = results
 
+    # flake8: noqa: C901
+    @staticmethod
+    def create(
+        task_specifications: Union[
+            Union[Circuit, Problem, Program, AnalogHamiltonianSimulation],
+            List[Union[Circuit, Problem, Program, AnalogHamiltonianSimulation]],
+        ],
+        delegate,  # noqa
```

Comment:
Why is the `# noqa` needed?

Reply:
Type hint `LocalSimulator` is creating a circular import

Reply:
Wouldn't the type be `BraketSimulator` though?

Reply:
Yes! My bad. Would it be BraketLocalSimulator or BraketSimulator ?